snuba/util.py

```diff
@@ -146,6 +150,9 @@ def column_expr(dataset, column_name, body, alias=None, aggregate=None):
         return complex_column_expr(dataset, column_name, body)
     elif isinstance(column_name, str) and QUOTED_LITERAL_RE.match(column_name):
         return escape_literal(column_name[1:-1])
+    elif isinstance(column_name, (list, tuple)):
```

Comment:
I would move this case into complex_column_expr since this method is meant to deal with one individual column.